### DISCUSSION TITLE
Link external library to a different accout

### DISCUSSION THREAD

**Discussion by Godd67** [immich-app/immich#6732] [Q&A] [answered]
Initially I set External path to my user and shared all Eternal library to my wife's account. She is now able to see all photos in her timeline however she has nothing under Explore so she cannot search by places or people. 
Now I want to link external library to her account and unlink mine, but I cant figure how can I safely do it.
![image](https://github.com/immich-app/immich/assets/1334713/9681f35d-c92d-4be7-bb3b-2035b0aa1071)
I want this external library to be attached to my wife account and shared with me.

**Comment by schuhbacca** [comment]
Delete library will not delete the underlying photos. Immich doesn't write or delete from external libraries. You should mount your library as :ro for safety though. So you should be able to just delete from your account and add to hers

**Comment by Godd67** [comment]
I tried to add External library to her account without deleting from mine and ran scan, but dont see any jobs started ( had to login back to my admin account to see jobs). Is it because immich already scanned files under mine account? Do I need to force scan in order to show photos in her timeline?

**Comment by Godd67** [comment]
I see this in server logs-
[Nest] 8  - 01/29/2024, 7:47:32 PM    WARN [AppGuard] Denied access to admin only route: /api/jobs

Does it mean she needs to be an admin to add External libraries?

**Comment by Godd67** [comment, marked as answer]
My bad, was a stupid typo in compose file. Everything is working now.
